ci/Jenkinsfiles/build.groovy

```diff
@@ -238,10 +233,14 @@ pipeline {
           Registry: ${DOCKER_REGISTRY}
           """
           script {
-            def moduleDir="docker/nuxeo-explorer-docker"
+            def moduleDir = 'docker/nuxeo-explorer-docker'
+            def nuxeoImageVersion = sh(returnStdout: true, script: 'mvn help:evaluate -Dexpression=nuxeo.platform.version | grep -v [INFO]').trim()
```

Comment:
apparently no :) i struggled a bit to parse that result, it seems to do the job as is.

Reply:
`grep -v [INFO]` will be interpreted as "exclude all lines containing I or N or F or O so it's not what you want.

Reply:
alright :+1:

Reply:
grepping `\[INFO\]` does not work as expected, nor `'\[INFO\]'` and i really which the mvn call did not force me to grep the output, so i'll replace it with simply "`INFO`" unless you have another suggestion: what am i missing?

Reply:
There're options for that in recent versions of help plugin:
```
mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate -Dexpression=project.version -q -DforceStdout
```
It may worth upgrading the plugin in Nuxeo, but don't know for what it is used.

Otherwise, `readMavenPom().getProperties().getProperty("nuxeo.platform.version")` from the groovy script may work 🤔.

Reply:
readMavenPom does not evaluate the expression, that's why i used the maven plugin, and i had tried the command line you gave: from my experience, using -q has always hidden the output anyway (hence the head scratching to understand how people use that command in scripts).
I've spent quite some time on this now, so unless someones thinks we'll be using versions "1.2.3-INFO" i think i'll go with the simple "INFO" inverse grepping...

Reply:
Just `INFO` is probably good enough for now.
The issue with `\[` is that you have many levels of parsing, and therefore escaping, in this line (at least groovy string, shell, and grep, each have escape syntaxes), and each parser probably needs its own escaping. So you may need two or four or maybe even eight backslashes.

Reply:
I've done the replacement (i confirmed the misbehaviour with "11.3-SNAPSHOT" version when keeping the `[` and `]`.
I'm leaving this conversation open if someone sees the issue with "\[INFO\]" or "'[INFO]" (tried `-F` and `-E` options too...)

Reply:
I've just tested the command on my computer and it works:

```
[~/ws/nuxeo-explorer (master u=)]
$ mvn org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate -Dexpression=nuxeo.platform.version -q -DforceStdout
11.3.50
```
Indeed -q does hide the output that's why they added -DforceStdout in 3.1.0

Of course, we can grep things if it's simpler.

Reply:
I wish it did work for me, sadly it's not the case: output is empty on my box.

Reply:
(Resolving with `grep -v INFO` for now)

Reply:
Used your solution after all Kevin, as Antoine confirmed it's working on https://github.com/nuxeo/nuxeo-customer-project-sample/pull/2 (it still does not work on my box but i'm not executing it on my box after all, and i had to `grep -v INFO | grep -v WARNING` after last tests, so it was starting to be too ugly)